packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -96,6 +96,39 @@ const toCSSDeclaration = (key: string, result: CSSOutput) => {
   return result.css.map((x) => ({ ...x, css: `${kebabCase(key)}: ${getItemCss(x)};` }));
 };
 
+/**
+ * Will return the variable declarator value node from meta's own path.
+ * It will compare the variable declarator node id's name with passed node name and
+ * returns its stringified value along with the node.
+ *
+ * Eg. If passed node is an Identifier with name `abcd` and somewhere in own path
+ * we have any variable declarator `const abcd = obj.value`, it will return
```

Comment:
nit: can we break this out into input outputs to make it read a little easier?

e.g.

```
Input: 
`const abcd = obj.value`

Output:
`{ variableName: '???', expression: '???'  }`
```